docker/openemr/5.0.2/php.ini

```diff
@@ -1337,22 +1353,23 @@ session.save_handler = files
 ; where MODE is the octal representation of the mode. Note that this
 ; does not overwrite the process's umask.
 ; http://php.net/session.save-path
-;session.save_path = "/var/lib/php/sessions"
+;session.save_path = "/tmp"
```

Comment:
@danehrlich1 ,
As above, this change seems really odd and wonder if this php config script did not come from alpine php??